doc/obs/todo.md

```
@@ -0,0 +1 @@
+multiple gpu https://enccs.github.io/gpu-programming/8-multiple_gpu/
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)